### PR TITLE
Add koin-android dependency

### PR DESCRIPTION
**Pull request by TheCodingWarlock**
I've added the koin-android dependency that was needed in the sample module to resolve the errors.

### CODE REVIEW

**Comment by gpeal** [comment]
@TheCodingWarlock What errors were you receiving?

**Comment by TheCodingWarlock** [comment]
```text
e: MvRx\sample\src\main\java\com\airbnb\mvrx\sample\core\MvRxApplication.kt: (7, 25): Unresolved reference: ext
e: MvRx\sample\src\main\java\com\airbnb\mvrx\sample\core\MvRxApplication.kt: (8, 17): Unresolved reference: dsl
e: MvRx\sample\src\main\java\com\airbnb\mvrx\sample\core\MvRxApplication.kt: (9, 17): Unresolved reference: dsl
e: M:\Code\ContributingProjects\MvRx\sample\src\main\java\com\airbnb\mvrx\sample\core\MvRxApplication.kt: (16, 39): Unresolved reference: Module
e: MvRx\sample\src\main\java\com\airbnb\mvrx\sample\core\MvRxApplication.kt: (16, 48): Expression 'applicationContext' of type 'Context!' cannot be invoked as a function. The function 'invoke()' is not found
e: MvRx\sample\src\main\java\com\airbnb\mvrx\sample\core\MvRxApplication.kt: (17, 9): Unresolved reference: bean
e: MvRx\sample\src\main\java\com\airbnb\mvrx\sample\core\MvRxApplication.kt: (30, 9): Unresolved reference: startKoin
e: MvRx\sample\src\main\java\com\airbnb\mvrx\sample\features\dadjoke\DadJokeDetailFragment.kt: (20, 25): Unresolved reference: ext
e: MvRx\sample\src\main\java\com\airbnb\mvrx\sample\features\dadjoke\DadJokeDetailFragment.kt: (52, 70): Unresolved reference: inject
e: MvRx\sample\src\main\java\com\airbnb\mvrx\sample\features\dadjoke\DadJokeIndexViewModel.kt: (14, 25): Unresolved reference: ext
e: MvRx\sample\src\main\java\com\airbnb\mvrx\sample\features\dadjoke\DadJokeIndexViewModel.kt: (55, 70): Unresolved reference: inject
e: MvRx\sample\src\main\java\com\airbnb\mvrx\sample\features\dadjoke\RandomDadJokeFragment.kt: (19, 25): Unresolved reference: ext
e: MvRx\sample\src\main\java\com\airbnb\mvrx\sample\features\dadjoke\RandomDadJokeFragment.kt: (38, 70): Unresolved reference: inject
```

That was the error I was getting @gpeal . But come to think of it, I would have just opened up an issue 😅. I thought that since the error was in the sample module I would just go ahead.

**Comment by TheCodingWarlock** [comment]
Plus the git history now doesn't look well documented now. Any tips on how to avoid commit messages like Merge remote-tracking branch 'branch' into 'branch'? I would really like some tips so that I can contribute to more open source projects in a better way. Thanks

**Comment by gpeal** [comment]
@TheCodingWarlock I will "squash and merge" so the final history will look clean.

**Comment by gpeal** [comment]
@TheCodingWarlock The app builds fine for me. What gave you that error?

**Comment by TheCodingWarlock** [comment]
I got the error in Android Studio while doing a gradle sync. Doing the
build on the terminal without adding the dependency also causes the error.
It got fixed after adding the dependency.

**Comment by gpeal** [comment]
@TheCodingWarlock What command and what version of Android Studio? I'm not getting the same thing.

**Comment by TheCodingWarlock** [comment]
Android Studio 3.3.2
Windows 10 Pro
The command I used is *gradlew build*
Try deleting the build directories and import the project again. Perhaps
the error might pop up. It does from my side.

**Comment by gpeal** [comment]
@TheCodingWarlock `./gradlew clean build` build successfully for me. Are you sure you don't have local changes or something?
CI also builds the app and it's green.

**Comment by TheCodingWarlock** [comment]
Maybe it's just on my side. I still keep getting the error after a clean build.

**Comment by piotrek1543** [comment]
@TheCodingWarlock seems that Gradle still keeps old build caches. I suggest you to stop all gradle daemons (I use `./gradlew --stop`), close Androd Studio and manually remove Gradle caches. 

In MacOSX it's in your home directory in folder `.gradle`. To see this folder use shortcut `CMD + Shift + .` to show/hide folders.

Hope it will help

File: sample/build.gradle [hunk 1]
```diff
@@ -48,6 +48,7 @@ dependencies {
     implementation "com.squareup.moshi:moshi-kotlin:$moshiVersion"
     kapt "com.squareup.moshi:moshi-kotlin-codegen:$moshiVersion"
     implementation "org.koin:koin-android-architecture:$koinVersion"
+    implementation "org.koin:koin-android:$koinVersion"
```
Comment (piotrek1543) [inline sample/build.gradle]: I wonder  if the dependency for Koin `ext.koinVersion = '0.9.3'` should be updated to the latest `1.0.2`.

**Pull request closed without merging**